Core/Exporters/IExporter.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.IO;
+
+namespace CKAN.Exporters
+{
+    /// <summary>
+    /// Represents an object that can 
```

Comment:
... do whatever it puts its mind to because it's beautiful and special.

I'll fix that.